### PR TITLE
Genarate traininglist.json for extendedfg.

### PR DESCRIPTION
**Pull request by yaochaorui**

### CODE REVIEW

**Review by innerlee — APPROVED** [review]

File: tools/data/matting/comp1k/extend_fg.py [hunk 1]
```diff
@@ -112,16 +115,14 @@ def main():
         os.makedirs(p, exist_ok=True)
 
     fg_names = osp.join(dir_prefix, f'{fname_prefix}_fg_names.txt')
+    save_json_path = f'{fname_prefix}_list.json'
```
Comment (innerlee) [inline tools/data/matting/comp1k/extend_fg.py:118]: Will this override the DIM filelist?

Comment (yaochaorui) [reply tools/data/matting/comp1k/extend_fg.py:118]: Yes.

Comment (innerlee) [reply tools/data/matting/comp1k/extend_fg.py:118]: Change the filename to `f'{fname_prefix}_list_fba.json'`

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
Oh the file structure should be modified accordingly

**Pull request merged**